File: 2643-row-with-maximum-ones/2643-row-with-maximum-ones.cpp
class Solution {
public:
    vector<int> rowAndMaximumOnes(vector<vector<int>>& mat) 
    {
        
        //o(m*n)
        int row_num=0;
        int num_ones=0;
        for (int i=0;i<mat.size();i++)
        {
            int temp_count=0;
            for (int j=0;j<mat[0].size();j++)
            {
                if (mat[i][j]==1){temp_count++;}
            }
            if (temp_count>num_ones)
            {
                row_num=i;
                num_ones=temp_count;
            }
        }

        return {row_num,num_ones};
        
    }
};

File: 2643-row-with-maximum-ones/2643-row-with-maximum-ones.java
class Solution {
    public int[] rowAndMaximumOnes(int[][] mat) {
        int m=mat.length;
        int n=mat[0].length;
        int count=Integer.MIN_VALUE;
        int a=0;
        for(int i=0;i<m;i++){
            int t=0;
            for(int j=0;j<n;j++){
                t+=mat[i][j];
                if(t>count){
                    count=t;
                    a=i;
                }
            }
        }
        return new int[]{a,count};
    }
}